<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ConfigureWeb</class>
 <widget class="QWidget" name="ConfigureWeb">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>2280</width>
    <height>561</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="accessibleName">
   <string>Online Play</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <item>
      <widget class="QGroupBox" name="groupBoxWebConfig">
       <property name="title">
        <string>yuzu Web Service</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayoutYuzuWebService">
        <item>
         <widget class="QLabel" name="web_credentials_disclaimer">
          <property name="text">
           <string>This token is for hosting public rooms in a lobby and are automatically generated the first time you save settings. To change your username, rename or switch the profile. If you are currently connected to multiplayer, exit and restart the emulator for changes to apply.</string>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QGridLayout" name="gridLayoutYuzuUsername">
          <item row="2" column="3">
           <widget class="QPushButton" name="button_reset_token">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="layoutDirection">
             <enum>Qt::RightToLeft</enum>
            </property>
            <property name="text">
             <string>Reset Token</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1" colspan="3">
           <widget class="QLabel" name="username"/>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="label_token">
            <property name="text">
             <string>Token: </string>
            </property>
           </widget>
          </item>
          <item row="1" column="4">
           <widget class="QLabel" name="label_token_icon"/>
          </item>
          <item row="0" column="0">
           <widget class="QLabel" name="label_username">
            <property name="text">
             <string>Username: </string>
            </property>
           </widget>
          </item>
          <item row="1" column="1" colspan="3">
           <widget class="QLineEdit" name="edit_token">
            <property name="maxLength">
             <number>80</number>
            </property>
           </widget>
          </item>
          <item row="2" column="2">
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_disable_info">
       <property name="text">
        <string>Web Service configuration can only be changed when a public room isn't being hosted.</string>
       </property>
       <property name="wordWrap">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QGroupBox" name="discord_group">
     <property name="title">
      <string>Discord Presence</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_21">
      <item>
       <widget class="QCheckBox" name="toggle_discordrpc">
        <property name="text">
         <string>Show Current Game in your Discord Status</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
